vshard/cfg.lua

```diff
@@ -44,6 +44,12 @@ local function check_replicaset_master(master)
     end
 end
 
+local function cfg_check_rebalancer_mode(mode)
+    if mode ~= 'auto' and mode ~= 'manual' and mode ~= 'off' then
+        error('Only "auto" and "manual" are supported')
```

Comment:
It looks like the "off" option is missing from the error message. Or was it intentionally skipped?

Reply:
Fixed.